rotkehlchen/tests/unit/decoders/test_velodrome.py

```diff
@@ -0,0 +1,510 @@
+            asset=Asset(f'eip155:10/erc20:{WETH_OP_POOL_ADDRESS}'),
+            balance=Balance(FVal('0.172627291949741834')),
+            location_label=user_address,
+            counterparty=CPT_VELODROME,
+            address=ZERO_ADDRESS,
+            product=EvmProduct.POOL,
+            notes=f'Receive 0.172627291949741834 vAMMV2-WETH/OP after depositing in velodrome pool {WETH_OP_POOL_ADDRESS}',  # noqa: E501
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('optimism_accounts', [['0x78C13393Aee675DD7ED07ce992210750D1F5dB88']])
+def test_remove_liquidity(optimism_transaction_decoder, optimism_accounts):
+    """Check that removing liquidity from a velodrome pool is properly decoded."""
+    tx_hex = deserialize_evm_tx_hash('0x81351bf9ca6d78bff92d2b714d68dd7c785bb70de0e250bc7bc6ab073736d1ce')  # noqa: E501
+    evmhash = deserialize_evm_tx_hash(tx_hex)
+    user_address = optimism_accounts[0]
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=optimism_transaction_decoder.evm_inquirer,
+        database=optimism_transaction_decoder.database,
+        tx_hash=tx_hex,
+    )
+    timestamp = TimestampMS(1694677251000)
+    expected_events = [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.OPTIMISM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(FVal('0.000024369543627752')),
+            location_label=user_address,
+            counterparty=CPT_GAS,
+            notes='Burned 0.000024369543627752 ETH for gas',
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=34,
+            timestamp=timestamp,
+            location=Location.OPTIMISM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.RETURN_WRAPPED,
+            asset=Asset(f'eip155:10/erc20:{WETH_OP_POOL_ADDRESS}'),
+            balance=Balance(FVal('0.086313645974870917')),
+            location_label=user_address,
+            counterparty=CPT_VELODROME,
+            address=WETH_OP_POOL_ADDRESS,
+            product=EvmProduct.POOL,
+            notes='Return 0.086313645974870917 vAMMV2-WETH/OP',
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=36,
+            timestamp=timestamp,
+            location=Location.OPTIMISM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.REMOVE_ASSET,
+            asset=A_WETH_OPT,
+            balance=Balance(FVal('0.002487103206849621')),
+            location_label=user_address,
+            counterparty=CPT_VELODROME,
+            address=WETH_OP_POOL_ADDRESS,
+            product=EvmProduct.POOL,
+            notes=f'Remove 0.002487103206849621 WETH from velodrome pool {WETH_OP_POOL_ADDRESS}',
```

Comment:
ok, indeed it would be better that this is consistent. Maybe its better to do this in a new PR because it will affect a lot of unrelated code for uniswap, curve etc.

Reply:
yes don't do it in this PR. We can adjust notes style in subsequent PR. Not sure exactly what style yabir means here but sure can be donelater. Keep this PR simple.